cargo-shuttle/src/args.rs

```diff
@@ -188,6 +188,12 @@ pub struct LoginArgs {
     pub api_key: Option<String>,
 }
 
+#[derive(Parser, Clone, Debug)]
+pub struct LogoutArgs {
+    /// Invalidate the API key before logging out
+    #[arg(long)]
+    pub invalidate_api_key: bool,
```

Comment:
We were discussing these things internally.
I agree that 'reset' sounds a bit better.
We decided to go for a flag instead of a subcommand in order to not flood the list of subcommands too much. Another idea we had was to have some sort of `cargo shuttle keys` command, with potential for expanding to more than 1 key, different types of keys, etc.
Do you know if the team has future considerations like this?